plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -776,9 +780,9 @@ typeWildCardActionTests = testGroup "type wildcard actions"
         doc <- createDoc "Testing.hs" "haskell" content
         _ <- waitForDiagnostics
         actionsOrCommands <- getAllCodeActions doc
-        let [addSignature] = [action | InR action@CodeAction { _title = actionTitle } <- actionsOrCommands
-                                    , "Use type signature" `T.isInfixOf` actionTitle
-                            ]
+        [addSignature] <- pure [action | InR action@CodeAction { _title = actionTitle } <- actionsOrCommands
+                                       , "Use type signature" `T.isInfixOf` actionTitle
+                               ]
```

Comment:
`findCodeActionsByInfix`?

Reply:
This case is meaningfuly different - here we're not specifying range in which actions should be find, but getAllCodeActions. So the potential for reuse of findCodeActionsByInfix.

Reply:
There is some potential for more reuse by defining better helper functions are reusing them more widely, but findCodeActionsByInfix similar to existing "ByPrefix" wouldn't be reusable because
- the place that has 2 infixes needs && but the helper function does ||
- one of the places uses getAllCodeActions and doesn't specify Range
- so I'd prefer not introducing it in this PR.